src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -3111,7 +3112,7 @@ void WriteErrorHelper(Exception exception)
                             RemoveFileSystemItem(file, force);
                         }
 
-                        if (_totalFiles > 0)
+                        if (_removeStopwatch.Elapsed.TotalSeconds > 0.2 && _totalFiles > 0)
                         {
                             _removedFiles++;
```

Comment:
1. This shouldn't be skipped while _removeStopwatch.Elapsed.TotalSeconds <= 0.2
2. I'd make 0.2 C# const since it is used twice.
3. I think 0.2 too small for interactive scenario. I believe user can wait 3-5 seconds before start worrying.

Reply:
Am i understanding correctly, that you want me to make it so that instead of all of it being skipped, the calcuations continue while the instance of the progressBar is created only after 3 - 5 seconds elapsed?

Reply:
Yes. 
We should do `_removedFiles++` while we don't show progress bar.

Reply:
Got it, i'll send another pull with your suggestions

Reply:
Maybe start with 3 seconds?